pkg/controller/job/job_controller_test.go

```diff
@@ -165,9 +165,23 @@ func setPodsStatuses(podIndexer cache.Indexer, job *batch.Job, pendingPods, acti
 		podIndexer.Add(pod)
 	}
 	for _, pod := range newPodList(succeededPods, v1.PodSucceeded, job) {
```

Comment:
should we move the code that adds the container statuses to `newPodList` instead?

Reply:
Done